code/game/objects/effects/decals/Cleanable/tracks.dm

```diff
@@ -199,7 +201,8 @@ var/global/list/image/fluidtrack_cache=list()
 			state=going_state
 			truedir=truedir>>4
 		var/icon/add = icon('icons/effects/fluidtracks.dmi', state, truedir)
-		add.SwapColor("#FFFFFF",track.basecolor)
```

Comment:
`add.color = track.color`
The color value seems to be fine but I can't varedit the overlays so not sure what's up there